services/QuillLMS/app/workers/admin_dashboard/admin_account_created_email_worker.rb

```diff
@@ -8,6 +8,17 @@ def perform(user_id, admin_user_id, school_id, is_reminder)
     @admin_name = User.find_by(id: admin_user_id)&.name
```

Comment:
I use instance variables to share values across methods in an instance of a class.  Otherwise, I use regular local variables if the scope should only apply to the method scope. (see some discussion [here](https://stackoverflow.com/a/10280944)).  In most/all of these worker classes the variable assignment is local to `perform` and then sent to a mailer.   Contrast this with the mailer classes where the instance variables are picked up by the [mailer].html.erb files indirectly (i.e. we don't pass them directly to anything).

For this reason, I would change all of the instance variables in all of these worker classes to just regular local variables.  For example:

```
    def perform(user_id, admin_user_id, school_id, is_reminder)
    user = User.find_by(id: user_id)
    admin_name = User.find_by(id: admin_user_id)&.name
    school_name = School.find_by(id: school_id)&.name
    is_reminder = is_reminder

    return unless user && admin_name && school_name

    user.mailer_user.send_admin_dashboard_admin_account_created_email(admin_name, school_name, is_reminder)

    analytics = SegmentAnalytics.new
    analytics.track_school_admin_user(
      user,
      SegmentIo::BackgroundEvents::ADMIN_CREATED_SCHOOL_ADMIN_ACCOUNT,
      school_name,
      admin_name
    )
  end
```

Reply:
That makes sense! Thanks for the explanation and link to the SO discussion 🙏